src/frontend/src/index.ts

```diff
@@ -10,6 +10,7 @@ import { checkRequiredFeatures } from "./utils/featureDetection";
 import { showWarningIfNecessary } from "./banner";
 import { displayError } from "./components/displayError";
 import { Connection } from "./utils/iiConnection";
+import { anyFeatures, features } from "./features";
```

Comment:
nth: following of course an opinionated feedback but, since `any` is the keyword that must absolutely be avoided, I would rather like a variable name that does not contain it 😁

Reply:
Gotcha! Will know better next time, although the variable was introduced a few months ago (i.e. not in this PR).